samples/Samples/Areas/Identity/Pages/Account/ConfirmAccount.cshtml.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Amazon.AspNetCore.Identity.Cognito;
+using Amazon.Extensions.CognitoAuthentication;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace Samples.Areas.Identity.Pages.Account
+{
+    [AllowAnonymous]
+    public class ConfirmAccountModel : PageModel
+    {
+        private readonly CognitoUserManager<CognitoUser> _userManager;
+
+        public ConfirmAccountModel(UserManager<CognitoUser> userManager)
+        {
+            _userManager = userManager as CognitoUserManager<CognitoUser>;
+        }
+
+        [BindProperty]
+        public InputModel Input { get; set; }
+
+        public string ReturnUrl { get; set; }
+
+        public class InputModel
+        {
+            [Required]
+            [Display(Name = "Code")]
+            public string Code { get; set; }
+        }
+
+        public void OnGet(string returnUrl = null)
+        {
+            ReturnUrl = returnUrl;
+        }
+
+        public async Task<IActionResult> OnPostAsync(string returnUrl = null)
+        {
+            returnUrl = returnUrl ?? Url.Content("~/");
+            if (ModelState.IsValid)
+            {
+                var userId = User.Claims.FirstOrDefault(c => c.Type == ClaimTypes.Name).Value;
+
+                var user = await _userManager.FindByIdAsync(userId).ConfigureAwait(false);
+                if (user == null)
+                {
+                    return NotFound($"Unable to load user with ID '{userId}'.");
+                }
+
+                var result = await _userManager.ConfirmSignUpAsync(user, Input.Code, true).ConfigureAwait(false);
+                if (!result.Succeeded)
+                {
+                    throw new InvalidOperationException($"Error confirming account for user with ID '{userId}':");
+                }
+                else
+                {
+                    if (returnUrl != null)
```

Comment:
return returnUrl != null ? LocalRedirect(returnUrl) : Page();

Reply:
LocalRedirect() and Page() do not return the same type